Application/EdFi.Ods.Admin.Api/Features/Applications/AddApplicationModelValidator.cs

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Ods.AdminApp.Management.Database.Commands;
+using FluentValidation;
+
+namespace EdFi.Ods.Admin.Api.Features.Applications
```

Comment:
I imported this from AdminApp.Web. Not entirely sure if it is needed. There are several other validators like that.

Earlier today, when I ran the API and swagger, I was getting 500 errors when I sent invalid data. The response message had a proper validation message, but it also had a stack trace. I have not tested to see if that is happening in the `main` branch as well.

Reply:
This class isn't being used. I need to delete it, but should first confirm that the validations are applied correctly in a different place.

Reply:
@CSR2017 FYI